Minnet App/src/components/molecules/TypographyIcon/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box } from "@mui/material";
+import React from "react";
+import IconAtom from "../../atoms/Icon";
+import TypographyMolecule from "../TypograpyMolecule";
+import { styled } from "@mui/system";
+import TypographyAtom from "../../atoms/Typography";
+import { theme } from "../../../theme/theme";
+import StarIconAtom from "../../atoms/StarIcon";
+
+interface TypographyIcon_Props {
+  src: string;
+  name: string;
+  subtitle: string;
+  price: string;
+  Change: string;
+  MarketCap: string;
+  isWatched: boolean;
+  onWatchToggle: () => void;
+}
+
+const StyledBox = styled(Box)(() => ({
+  width: "1100px",
+  height: "74px",
+  display: "grid",
+  gridTemplateColumns: "300px 250px 200px 200px 100px",
+  alignItems: "center",
+  backgroundColor: theme.palette.primary.main,
+  border: "1px solid rgba(232, 232, 247, 1)",
+  marginBottom: "5px",
+  padding: "0 20px",
+}));
+
+const StarBox = styled(Box)(() => ({
+  display: "flex",
+  justifyContent: "center",
+  width: "100%",
+}));
+
+const StyledIconTypographyBox = styled(Box)(() => ({
+  display: "flex",
+  gap: "1rem",
+  alignItems: "center",
+}));
+
+export const TypographyIcon: React.FC<TypographyIcon_Props> = ({
+  ...props
```

Comment:
The current name doesn't accurately reflect the purpose of the component. Consider using a more descriptive and relevant name, such as TradeRow or something similar